services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime.date,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: datetime.date = None) -> QuerySet:
+    movies_sessions = MovieSession.objects.all()
+
+    if session_date:
+        movies_sessions = movies_sessions.filter(show_time__date=session_date)
+
+    return movies_sessions
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime.date = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> None:
+    movie_session = MovieSession.objects.all().filter(id=session_id)
+
+    if show_time:
+        movie_session.update(show_time=show_time)

```

Comment:
It would be more efficient in terms of database interactions to get a `MovieSession` object, set required fields via Python object interface, and call `obj.save()` afterwards.